models/_sector/dex/trades/_schema.yml

```diff
@@ -8,6 +8,7 @@ models:
       contributors: 0xRob, hosuke, jeff-dude, tomfutago
```

Comment:
```suggestion
      contributors: 0xRob, hosuke, jeff-dude, tomfutago
      long_description: '{{ doc("dex_trades_doc") }}'
```

Reply:
this doesn't work, tried to compile this in these configs:
- `'{{ doc("dex_trades_doc") }}'`
- `{{ doc("dex_trades_doc") }}`
- "{{ doc("dex_trades_doc") }}"

ChatGPT also doesn't know of any workaround, we don't have access to the `doc` function within the `meta` context.